app/controllers/album.js

```diff
@@ -0,0 +1,32 @@
+const albumService = require('../services/albumService');
+const log = require('../logger');
+
+module.exports = {
+  getAlbums: (req, res, next) => {
+    albumService
+      .getAlbums()
+      .then(response => response.json())
+      .then(albums => {
+        res.status(200).send(albums);
+      })
+      .catch(err => {
+        log.error(err.message);
+        err.internalCode = 'CONECTION_ERROR';
+        next(err);
+      });
+  },
+
+  getPhotoOfAlbum: (req, res, next) => {
+    const albumId = req.params.id;
+    albumService
```

Comment:
Warning: if you not returns promises you can expose yourself to intermittences in your functions. Your function can end but the promise can are pending